Add release notes for v.1.13.0 (#2078)
SAP/luigi: blog/2021-06-07-release-notes.md

---
title: Luigi v1.13.0
seoMetaDescription: Release notes for Luigi v1.13.0
author:
  - Aleksandra Simeonova
layout: blog
---

> Luigi 1.13.0 is the latest Luigi release. It includes updates of Fundamental Styles, new options in the user settings, as well as other features and bugfixes. Read the release notes below to learn more.

Reviewer: ```suggestion
Luigi 1.13.0 is the latest Luigi release. It includes updates of Fundamental Library Styles, new options in the user settings, as well as other features and bug fixes. Read the release notes below to learn more.
```

<!-- Excerpt -->

> #### Breaking changes in Fundamental Styles

Reviewer: ```suggestion
#### Breaking changes in Fundamental Library Styles
```

> With Luigi v1.13.0, the new v0.17.0 of Fundamental Library Styles was included. It led to having breaking changes in the Luigi Dialog/Modal layout and Shellbar Counter. New HTML attributes were added to those components to increase accessibility. You can see the new layout of these components here: [Dialog](https://sap.github.io/fundamental-styles/?path=/docs/components-dialog--default-dialog) and [Shellbar Counter](https://sap.github.io/fundamental-styles/?path=/docs/components-shellbar--primary).

Reviewer 2: Maybe we could add sth. like
"As always this does only affect you if you have customized fundamentals css in your app" or sth. like that

#### Extend options of enum type in user settings

> In Luigi's [user settings](https://docs.luigi-project.io/docs/user-settings), an `enum` array can now can store options objects. You can use them if you want to add more options in the user settings for the user to choose from. These objects need `value` and `label` as key. For example:

Reviewer: ```suggestion
In Luigi's [user settings](https://docs.luigi-project.io/docs/user-settings), an `enum` array can now store options objects. You can use them if you want to add more options in the user settings for the user to choose from. These objects need `value` and `label` as key. For example:
```
